### ISSUE TITLE
gradlew.bat is not recognized as an internal or external command

### ISSUE THREAD

**Issue by GrandSchtroumpf** [eskatos/gradle-command-action#3]
Hello,
When running the actions on `windows-latest` I've got this error : 
`'d:\a\{PROJECT_NAME}\{PROJECT_NAME}\gradlew.bat' is not recognized as an internal or external command`.

Here is my action file : 
```yml
name: Build

on:
  push:
    branches:
      - build

jobs:
  build:
    name: Build
    runs-on: windows-latest
    steps:
      - uses: actions/checkout@v1
      - uses: actions/setup-java@v1
        with:
          java-version: 11
      - run: cd android  # cd to the android folder
      - uses: eskatos/gradle-command-action@v1
        with:
          arguments: build
```

According to the [software environment documentation](https://help.github.com/en/github/automating-your-workflow-with-github-actions/software-in-virtual-environments-for-github-actions#gradle) Gradle in Windows Server 2019 is in the PATH. Why doesn't the action only run `gradlew task` ?

Thanks !

**Comment by eskatos** [comment]
Hi @GrandSchtroumpf, thanks for the report!

This action uses the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) from your sources by default. The error you get `gradlew.bat' is not recognized as an internal or external command` probably means that you don't have a Gradle Wrapper in your source repository.

I would recommend to check a Gradle Wrapper in your source repository.

Alternatively, you can specify a Gradle version to use in your workflow:

```yaml
 - uses: eskatos/gradle-command-action@v1
   with:
     gradle-version: 5.6.4
```

The reason this action doesn't use the Gradle installation provided on Github actions machines is that the Gradle version is unknown/changing and that would make builds non reproducible over time.

HTH

**Comment by GrandSchtroumpf** [comment]
Hi @eskatos, thanks for your feedbacks.
You're right the issue was that I wasn't on the right working directory. I tried to provide `build-root-directory` but it didn't work neither. I finally manage to make it work without this action like this : 
```yml
- run: ./gradlew build
  working-directory: ${{ github.workspace }}/android
```
I'll close the issue, I think it's more a lack of knowledge of Gradle & Github Actions from my side than an problem with the lib.
Thank you again for your time and explanation, it helped me a lot :)

**Comment by eskatos** [comment]
@GrandSchtroumpf glad you got it working.

In the meantime @jmfayard discovered a bug in the action that was triggered when using `build-root-directory` on windows where the path to the `gradlew` wrapper wasn't resolved correctly. I assume this is what you faced. I just released version `v1.0.2` of this action with a fix, you'll get the update by using the `v1` tag.

For your workflow the following should work now:

```yaml
 - uses: eskatos/gradle-command-action@v1
   with:
     build-root-directory: android
     arguments: build
```

**Comment by mbjelac** [comment]
> Alternatively, you can specify a Gradle version to use in your workflow: ...

We decided to go without the wrapper, as checking in a binary is not really our thing :) - the stable Gradle version is guaranteed by `gradle-version` and it works just as you specified here.

However, I wish I had found this piece of information in the main readme and didn't have to browse through issues. I found this quite by accident.

A section, or just a line about "using the action without Gradle wrapper" would be nice.
